raiden-ts/src/services/epics.ts

```diff
@@ -135,6 +136,7 @@ function fetchLastIou$(
         });
 
       const { last_iou: lastIou } = decode(LastIOUResults, losslessParse(text));
+      // accept last IOU only if it was signed by us
```

Comment:
Do you think the code is not clear enough about that?

Reply:
Just wanted to be double-sure, to avoid to carelessly clean this and miss the behavior which must be strict here for safety reasons.